### ISSUE TITLE
ModuleNotFoundError: No module named 'sqlalchemy.orm.dependency'

### ISSUE THREAD

**Issue by kerwinxu** [Nuitka/Nuitka#2227]
Before submitting an Issue, please review the [Issue Guidelines](https://github.com/Nuitka/Nuitka/blob/develop/CONTRIBUTING.md#submitting-an-issue).

* Please check whether the bug was already reported or fixed.

* Please check out if the develop version of Nuitka works better for you.

  Download source, packages [from here](https://nuitka.net/doc/download.html)
  where you will also find instructions how to do it via pip or git.

If you want to post a problem/bug, to help us understand and resolve your issue
please check that you have provided at least the information below, and discard
up to here:

* Nuitka version, full Python version, flavor, OS, etc. as output by *this*
  command.

  > python -m nuitka --version
**1.5.3
Commercial: None
Python: 3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)]
Flavor: CPython Official
Executable: C:\Users\kerwin\AppData\Local\Programs\Python\Python310\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
Version C compiler: d:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.35.32215\bin\HostX64\x64\cl.exe (cl 14.3).**

  If you are not providing the full output. The issue cannot be solved and will
  be considered `invalid`. It does more than you think, and we are adding more
  all the time. We do not want to left guessing or trying things out to
  reproduce your issue.

* How did you install Nuitka and Python

  Did you use pip, anaconda, deb, rpm, from source, git clone, then install into
  a virtualenv or not, this is very important usually and one thing, the above
  command does not tell us (yet).

**pip install nuitka** 

* The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g.
  `numpy`, you are expected to shorten this to the relevant ones.

  > python -m pip freeze
**altgraph==0.17.3
anyio==3.6.2
appdirs==1.4.4
argcomplete==1.10.3
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
asgiref==3.6.0
asttokens==2.2.1
async-generator==1.10
attrs==22.2.0
audioread==3.0.0
autopep8==2.0.1
Babel==2.12.1
backcall==0.2.0
beautifulsoup4==4.8.2
bleach==6.0.0
borb==2.1.12
bs4==0.0.1
certifi==2022.12.7
cffi==1.15.1
chardet==3.0.4
charset-normalizer==3.0.1
click==8.1.3
colorama==0.4.6
comm==0.1.2
compressed-rtf==1.0.6
contourpy==1.0.7
cryptography==40.0.2
cycler==0.11.0
dearpygui==1.9.1
debugpy==1.6.6
decorator==5.1.1
defusedxml==0.7.1
deprecation==2.1.0
distlib==0.3.6
distro==1.8.0
Django==4.1.7
docx2txt==0.8
docxcompose==1.4.0
docxtpl==0.16.6
ebcdic==1.1.1
et-xmlfile==1.1.0
exceptiongroup==1.1.0
executing==1.2.0
extract-msg==0.28.7
fastjsonschema==2.16.3
filelock==3.9.0
filetype==1.2.0
Flask==2.2.3
fonttools==4.38.0
fqdn==1.5.1
gitchangelog==3.0.4
greenlet==2.0.2
h11==0.14.0
idna==3.4
imageio==2.27.0
IMAPClient==2.1.0
iniconfig==2.0.0
ipykernel==6.21.2
ipython==8.10.0
ipython-genutils==0.2.0
ipywidgets==8.0.4
isoduration==20.11.0
itsdangerous==2.1.2
jedi==0.18.2
jieba==0.42.1
Jinja2==3.1.2
joblib==1.2.0
jsonpointer==2.3
jsonschema==4.17.3
jupyter==1.0.0
jupyter-console==6.6.2
jupyter-events==0.6.3
jupyter_client==8.0.3
jupyter_core==5.2.0
jupyter_server==2.3.0
jupyter_server_terminals==0.4.4
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.5
kiwisolver==1.4.4
lazy_loader==0.2
llvmlite==0.39.1
lxml==4.9.2
markdown-it-py==2.2.0
MarkupSafe==2.1.2
matplotlib==3.7.0
matplotlib-inline==0.1.6
mdurl==0.1.2
metakernel==0.29.4
mistune==2.0.5
modbus-tk==1.1.3
msgpack==1.0.5
nbclassic==0.5.2
nbclient==0.7.2
nbconvert==7.2.9
nbformat==5.7.3
nest-asyncio==1.5.6
networkx==3.1
notebook==6.5.2
notebook_shim==0.2.2
npTDMS==1.6.2
Nuitka==1.5.3
numba==0.56.4
numpy==1.23.5
octave_kernel==0.35.1
olefile==0.46
openpyxl==3.1.1
ordered-set==4.1.0
outcome==1.2.0
packaging==23.0
pandas==1.5.3
pandocfilters==1.5.0
parso==0.8.3
pdfminer3k==1.3.4
pefile==2023.2.7
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.4.0
pip-search==0.0.12
pipenv==2023.2.18
platformdirs==3.0.0
pluggy==1.0.0
ply==3.11
pooch==1.6.0
prometheus-client==0.16.0
prompt-toolkit==3.0.37
psutil==5.9.4
ptyprocess==0.7.0
pure-eval==0.2.2
pycodestyle==2.10.0
pycparser==2.21
pycryptodome==3.17
pygame==2.2.0
Pygments==2.14.0
pyinstaller==5.10.1
pyinstaller-hooks-contrib==2023.2
PyMuPDF==1.22.1
PyMySQL==1.0.2
pyparsing==3.0.9
pyperclip==1.8.2
pypng==0.20220715.0
PyQt5==5.15.9
PyQt5-Qt5==5.15.2
PyQt5-sip==12.12.0
pyrsistent==0.19.3
pyserial==3.5
PySide6==6.4.2
PySide6-Addons==6.4.2
PySide6-Essentials==6.4.2
PySocks==1.7.1
pytest==7.3.1
python-barcode==0.14.0
python-dateutil==2.8.2
python-docx==0.8.11
python-json-logger==2.0.7
python-pptx==0.6.21
pytz==2022.7.1
pytz-deprecation-shim==0.1.0.post0
PyWavelets==1.4.1
pywin32==305
pywin32-ctypes==0.2.0
pywinpty==2.0.10
PyYAML==6.0
pyzmq==25.0.0
QDarkStyle==3.1
qrcode==7.4.2
qt-material==2.14
qtconsole==5.4.0
QtPy==2.3.0
requests==2.28.2
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
rich==13.3.1
scikit-image==0.20.0
scikit-learn==1.2.1
scipy==1.10.1
selectivesearch==0.4
selenium==4.8.2
Send2Trash==1.8.0
shiboken6==6.4.2
six==1.12.0
sniffio==1.3.0
sortedcontainers==2.4.0
soundfile==0.12.1
soupsieve==2.4
soxr==0.3.5
SpeechRecognition==3.8.1
SQLAlchemy==2.0.9
sqlparse==0.4.3
stack-data==0.6.2
tabula-py==2.7.0
tenacity==8.2.2
terminado==0.17.1
textract==1.6.5
threadpoolctl==3.1.0
tifffile==2023.4.12
tika==2.6.0
tinycss2==1.2.1
tomli==2.0.1
torch==1.13.1
torchaudio==0.13.1
torchvision==0.14.1
tornado==6.2
tqdm==4.64.1
traitlets==5.9.0
trio==0.22.0
trio-websocket==0.9.2
typing_extensions==4.5.0
tzdata==2022.7
tzlocal==4.3
uri-template==1.2.0
urllib3==1.26.14
virtualenv==20.19.0
virtualenv-clone==0.5.7
Wand==0.6.11
wcwidth==0.2.6
webcolors==1.12
webencodings==0.5.1
websocket-client==1.5.1
Werkzeug==2.2.3
widgetsnbextension==4.0.5
wsproto==1.2.0
xlrd==1.2.0
XlsxWriter==3.1.0
yapf==0.32.0
zstandard==0.20.0**

* Many times when you get an error from Nuitka, your setup may be special

  Then even a `print("hello world")` program will not work, please try that and
  report that error instead. We do not need a report that says "Tensorflow does
  not work" when nothing actually works for you. You ought to also read the User
  Manual and check that your setup is actually supported already.

* Also supply a Short, Self Contained, Correct, Example

  That demonstrates the issue i.e a small piece of code which reproduces the
  issue and can be run with out any other (or as few as possible) external
  dependencies. Issues without this may will likely be rejected without much
  consideration. Often this can be as simple as importing a package, if this
  is a packaging issue, try that first.

  Pointers to repositories with usage of ``pipenv`` are very welcome, archives
  with examples are too. This must be only source code, binaries are not used
  ever by us. But beware, that e.g. we cannot click around and stuff. Still do
  an effort to make the usage obvious. Having a compile script in the repo is
  perfect.

* Provide in your issue the Nuitka options used

  Ideally use the `# nuitka-project:` options feature in the code, so options
  and example code go along. Alternatively state the command line.

  [Nuitka Options in the code](https://nuitka.net/doc/user-manual.html#nuitka-options-in-the-code)

* Avoid unnecessary options

  Do not use `--onefile` if the issue also happens with `--standalone`. Minimize
  the use of options as far as you can, please. Do not disable outputs with
  `--quiet` and do not disable warnings.

* Note if this is a regression

  If it used to work in an earlier version of Nuitka, please note what you know
  about that. Since git bisect is a thing for which we do a lot of error to make
  it usable, this will help dramatically to isolate the issue.

* Consider getting commercial support

  [Nuitka commercial](https://nuitka.net/doc/commercial.html) offers
  subscriptions and priority support. This will accelerate your problem solution
  and helps to sustain Nuitka development. Also you then have a chance to
  provide source code of your project to us, which might simplify things, or pay
  for time to solve your issues in your environment.

Some things are not welcome, please consider it.

* Do *not* post screenshots

  These are not welcome unless absolutely necessary, e.g. because of Qt display
  problem, instead capture the output of programs, so things are searchable and
  copy&paste will work. I just plainly don't want to manually copy strings and
  hope they match.

* Do *not* close the issue yourself, we will close things on stable releases

  We close issues only when they are released as a stable version, e.g. in a
  hotfix or a new release, before that it will be "Done" in planning and go
  through `factory` and `develop` tags to indicate they are solved there.

  Of course, if you find out your issue is invalid, please do close it, and
  we then attach the `invalid` tag.

* Do *not* report against factory version

  Unless you were asked to test it there, it is frequently very broken, and
  there is only noise to be had. Telling me about it on Discord would be a
  better idea.

* Do *not* let this template remain part of the issue, it's noise.

nuitka --standalone --mingw --nofollow-imports --show-memory --show-progress --plugin-enable=pyside6 --output-dir=../out main.py
 main.exe
Traceback (most recent call last):
  File "E:\onedrive\project\python\文档整理\DOCUME~1\out\MAIN~1.DIS\main.py", line 15, in <modul
  File "E:\onedrive\project\python\文档整理\DOCUME~1\out\MAIN~1.DIS\database.py", line 5, in <mo
dule database>
  File "E:\onedrive\project\python\文档整理\DOCUME~1\out\MAIN~1.DIS\sqlalchemy\orm\__init__.py",
 line 170, in <module sqlalchemy.orm>
  File "E:\onedrive\project\python\文档整理\DOCUME~1\out\MAIN~1.DIS\sqlalchemy\orm\__init__.py",
 line 166, in __go
  File "E:\onedrive\project\python\文档整理\DOCUME~1\out\MAIN~1.DIS\sqlalchemy\util\preloaded.py
", line 139, in import_prefix
**ModuleNotFoundError: No module named 'sqlalchemy.orm.dependency'**

**Comment by kayhayen** [comment]
The combination `--standalone --nofollow-imports` has this as an expected result.
